docs/application/dotnet/guides/user-interface/nui/flex-layout.md

```diff
@@ -1,3 +1,11 @@
+<style>
+    .tabcontent img {
+        border: 1px solid #555;
+        max-width: 100% !important;
+        max-height: 100%;
+    }
+</style>
+
 # Flex Layout
 
 `FlexLayout` is a flexible box layout, that provides a more efficient way to layout, align, and distribute space among items in the container, even when their size is unknown or dynamic.
```

Comment:
The comma is not necessary considering the information after that is a dependent clause. Ref: https://developers.google.com/style/commas#commas-separating-independent-from-dependent-clauses 
```suggestion
`FlexLayout` is a flexible box layout that provides a more efficient way to layout, align, and distribute space among items in the container, even when their size is unknown or dynamic.
```